fundingmanager.go

```diff
@@ -1547,6 +1548,37 @@ func (f *fundingManager) handleFundingAccept(fmsg *fundingAcceptMsg) {
 		UpfrontShutdown: msg.UpfrontShutdownScript,
 	}
 	err = resCtx.reservation.ProcessContribution(remoteContribution)
+
+	// The wallet has detected that a PSBT funding process was requested by
```

Comment:
Still getting through this, but we could possibly attempt to encapsulate more of this in the reservation struct itself.

Reply:
Or I guess the idea is that this method will end up being called _twice_ for PSBT based funding?

Reply:
One alternative would be to create a new goroutine for this specific purpose which will then resume things on its own, independent of the main funding manager control loop.

Reply:
I like the idea with the goroutine. I went down that road and implemented it. I didn't have time to fully test it yet but it should work like that. I'll continue on it tomorrow.